### DISCUSSION TITLE
Does PyArmor 8.5 Pro version support Pyarmor 9.0 Pro? Do I need to purchase 9.0 Pro again?

### DISCUSSION THREAD

**Discussion by ericedmund** [dashingsoft/pyarmor#2012] [Q&A]
I purchased the PyArmor 8.5 Pro version license in May, and now I want to use PyArmor 9.0 Pro version. Do I need to purchase it again?

**Comment by jondy** [comment]
Please check upgrade notes issue #1958

**Reply by ericedmund** [reply]
it works , Thanks so much!
